decompy/DataGathering/RepoFilter.py

```diff
@@ -40,46 +42,84 @@ def __init__(self, search, language=None, blacklist=None, per_page=100):
         self.language = language
         self.blacklist = blacklist
         self.per_page = per_page
+        self.username = username
+        self.password = password
 
-    def get_results(self, page):
+    def get_results(self, date, page):
         """
         Makes a single request to the GitGub api for a page with results matching
         the search criteria
 
+        :param date: because github only allows us 1000 results because they are bad at making an api.
+        :type; str
+
         :param self:
         :param page: Which page of results should be fetched
         :type: int
         """
 
         language_string = ""
         if self.language:
-            language_string = "language=%s&" % self.language
-
-        url = "https://api.github.com/legacy/repos/search/%s?%sstart_page=%d" % (
-            quote(self.search), language_string, page)
-        content = WebNavigator.getContent(url)
-        json_content = json.loads(content)
-        return json_content["repositories"]
-
-    def offline_results(self, filename, start_page, end_page):
+            language_string = "language:%s" % "C"
+
+        url = "https://api.github.com/search/repositories?q=%s%s+created:%s&page=%d" % (
+            quote(self.search), language_string, date, page)
+
+        print("searching for...", url)
+        # pageSource = ""
+        response = None
+        try:
+            # Download the zip of the repository

```

Comment:
Not entirely sure I understand why this class is now downloading the zip file for the repo instead of JJ's portion of the code.